app-ui5/webapp/Component.js

```diff
@@ -0,0 +1,8 @@
+sap.ui.define(["sap/ui/core/UIComponent"], function(UIComponent) {
+	"use strict";
+	return UIComponent.extend("sap.ui.demo.todo.Component", {
+		metadata: {
```

Comment:
Shall we add the "IAsyncContentCreation" Interface to this component so that we can remove the "async: true" from the "rootView" settings in manifest.json